docs/topics_5/web-create-project.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='add_project_{context}']
+= Creating a project
+
+You create a project using a wizard-like interface.
+
+.Procedure
+
+. In the navigation pane, click *Projects*.
+. On the *Projects* page, click *Create project*.
++
+.Create project
+image::web-add-project_51.png[Create project]
+
+. Enter a unique name for the project.
```

Comment:
. Enter the project name and an optional description and then click Next. 

That way you can get 3 steps in one.